lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Controller     = require('./controller');
+const MovieValidator = require('../../../validators/movie');
```

Comment:
nit: Our general structure for validators is `validators/movies/create.js` such that it's one file per validator.

I wouldn't bother changing it now until you make a second validator though.

Reply:
Good to know. I'll make the change anyways.

What is the semantic difference between naming a file create.js versus index.js?